agent/http.go

```diff
@@ -587,6 +587,12 @@ func (s *HTTPHandlers) Index(resp http.ResponseWriter, req *http.Request) {
 	// Check if this is a non-index path
 	if req.URL.Path != "/" {
 		resp.WriteHeader(http.StatusNotFound)
+
+		if strings.Contains(req.URL.Path, "/v1/") {
+			fmt.Fprint(resp, "Invalid URL path: not a recognized HTTP API endpoint")
+		} else {
+			fmt.Fprint(resp, "Invalid URL path: if attempting to use the HTTP API, ensure the path starts with '/v1/'")
```

Comment:
minor: maybe a trailing newline on these?
```suggestion
			fmt.Fprintln(resp, "Invalid URL path: not a recognized HTTP API endpoint")
		} else {
			fmt.Fprintln(resp, "Invalid URL path: if attempting to use the HTTP API, ensure the path starts with '/v1/'")
```

Reply:
Good catch, I'll make this change then merge